dateparser/freshness_date_parser.py

```diff
@@ -144,8 +144,10 @@ def get_kwargs(self, date_string):
         return kwargs
 
     def get_date_data(self, date_string, settings=None):
+        from dateparser.date import DateData
```

Comment:
Why is this here, to avoid a circular dependency?

Reply:
Yes. Maybe we can search for another workaround.

Reply:
Well, even though I do prefer to resolve it a different way, I think that can be done on a separate pull request later.